infrastructure/alerts.tf

```diff
@@ -37,7 +37,7 @@ module "fpl-grant-case-access-failure-alert" {
   app_insights_name          = "${var.product}-${var.component}-appinsights-${var.env}"
   alert_name                 = "${var.product}-grant-case-access-failure"
   alert_desc                 = "Grant case access failure"
-  app_insights_query         = "exceptions | where type contains 'GrantCaseAccessException' | project timestamp , outerMessage"
+  app_insights_query         = "exceptions | where type contains 'GrantCaseAccessException' | project timestamp, operation_Id, outerMessage"
```

Comment:
not sure if now we need this dedicated alert now, the only difference is severity. Now same exception is counted twice

Reply:
Yes i'd agree @tomaszpowroznik, only advantage we'd get is grouping I guess. @nkoranne thoughts on removing?

Reply:
@Jamesnelson117  and @tomaszpowroznik , I am happy to remove it as long as information in alert is visible quickly that 401 error has been reported.  Also agree with the point that this alert can differentiated based on severity.  We can think of adding dedicated alert in future if we see too may such type of exceptions